Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.22.5-alpine
+
+WORKDIR /devzat
+
+COPY . .
+
+RUN go build
+
+EXPOSE 8080
```

Comment:
The default port for the server is `2221`, and the alternative one is `8080`. Is there any specific reason why only the latter is specified? The `EXPOSE` instruction is only informative but still.

Reply:
During my tests i've found that I can connect to the chatroom using port 8080 so I replicated that in the dockerfile. I've spent no more than 15 minutes on the dockerfile + PR, no idea if other configurations works but interested in figuring it out

Reply:
there's a bunch of ports used. you'd probably want to expose the rpc one and also both the ssh ones. please have a look at the source